main.go

```diff
@@ -675,6 +765,15 @@ func sum(stats map[string]string, keys ...string) (float64, error) {
 	return s, nil
 }
 
+func hasError(errors ...error) error {
```

Comment:
This method would be more accurately named `firstError`. `hasError` indicates it would return a boolean.